lib/cc/analyzer/issue_validator.rb

```diff
@@ -5,6 +5,7 @@ class IssueValidator
         IssueCategoryValidation,
         IssueCheckNamePresenceValidation,
         IssueDescriptionPresenceValidation,
+        IssueLocationExistenceValidation,
```

Comment:
I slightly like using `presence` throughout here, instead of mixing with `existence`, but looks like we've already mixed a bit so, perhaps that's fine. WDYT?